tasks/uk/census/ons.py

```diff
@@ -34,12 +34,12 @@ def run(self):
         api_id = (meta['structure']['keyfamilies']['keyfamily'][0]['id']).lower()
 
         # Download for SA (EW,S) and OA (NI) in a single file
-        with self.output().temporary_path() as tmp, open(os.path.join(tmp, '{}.csv'.format(self.table)), 'w') as outcsv:
+        with self.output().temporary_path() as tmp, open(os.path.join(tmp, '{}.csv'.format(self.table)), 'wb') as outcsv:
             skip_header = False
             for geo in self.GEO_TYPES:
-                remote_file = urllib.urlopen(self.DOWNLOAD_URL.format(id=api_id, geo=geo))
+                remote_file = urllib.request.urlopen(self.DOWNLOAD_URL.format(id=api_id, geo=geo))
```

Comment:
Out of curiosity, how come you don't need `import urllib.request`?

Reply:
I'm importing `urllib`

Reply:
I think you need to import `urllib.request`. At least, I needed to for other changes. Anyway, if it works, it's good enough for me. Speaking about working, tests are broken because of the tools thing.

Reply:
Yes... the `tools` thing again. I'm changing it to avoid breaking the tests until the change eventually reaches `master`

Reply:
Anyway this PR doesn't solve anything related to the issue... yet